charts/lightstepsatellite/values.yaml

```diff
@@ -61,7 +90,7 @@ ingress:
 
 lightstep:
 
-  # REQUIRED: Input your Satellite API Key
+  # -- REQUIRED: your Satellite API Key - if not set, `lightstep.collector_satellite_key_secret_name` and `lightstep.collector_satellite_key_secret_key` must be set
```

Comment:
The assumption is that people will know to create the secret in their namespace beforehand?

Reply:
If people want to explicitly create their own secret in an environment, they can do so. This allows users to either put in their Sat key directly, or reference an existing secret with it.

Reply:
@ishg I'm not sure what the assumption of the authors of this section was/is; this is just a change to clarify that the chart requires either the key is provided or the secret name/key is defined and to add this to the description column in the README.